ckanext/dataset_subscriptions/tests/test_actions.py

```diff
@@ -44,22 +46,35 @@ def test_with_no_dataset_updates():
 
 
 @pytest.mark.usefixtures("clean_db")
+@pytest.mark.usefixtures("with_plugins")
 def test_with_dataset_updates():
     user=create_user_with_resources(with_activity=True)
     activity_list = helpers.call_action("dashboard_activity_list", context={"user": user["id"]})
     # We'll be returning only package notifications
     filtered_activity_list = [activity for activity in activity_list
                 if "package" in activity["activity_type"]]
     assert ["package" in activity["activity_type"] for activity in filtered_activity_list]
-    
 
 
 @pytest.mark.usefixtures("with_request_context", "clean_db")
 @pytest.mark.ckan_config("ckan.activity_streams_email_notifications", True)
+@pytest.mark.usefixtures("with_plugins")
 @mock.patch("ckan.logic.action.update.request")
 def test_send_notification_injection(mock_request):
     user=create_user_with_resources(with_activity=True)
     notifications = email_notifications.get_notifications(user, datetime.datetime.fromtimestamp(3600))
     # we expect two notifications - one for a new dataset and one for a resource
     assert [ "2 new activities from CKAN" in notification["subject"]  for notification
             in notifications ]
+
+
+@pytest.mark.usefixtures("with_request_context","clean_db")
+@pytest.mark.ckan_config("ckan.activity_streams_email_notifications", True)
+@pytest.mark.usefixtures("with_plugins")
+@pytest.mark.usefixtures("mail_server")
+@mock.patch("ckan.logic.action.update.request")
+def test_send_notification(mock_request):
```

Comment:
what does this test exactly do? Can you provide more descriptive name for it?

Reply:
done